cornucopia/src/lib.rs

```diff
@@ -70,15 +70,15 @@ pub fn generate_live(
 ///
 /// By default, the container manager is Docker, but Podman can be used by setting the
 /// `podman` parameter to `true`.
-pub fn generate_managed(
-    queries_path: &Path,
+pub fn generate_managed<P: AsRef<Path>>(
+    queries_path: P,
     schema_files: &[PathBuf],
```

Comment:
`load_schema` is now defined as 
```rust
load_schema<P: AsRef<Path>>(client: &mut Client, paths: &[P]) -> Result<(), Error>
```
So we should probably also take the schema_paths as `schema_files: &[P]` instead of `schema_files: &[PathBuf]`.